### PR TITLE
SUBMARINE-631. [WEB] Fix workbench notebook page

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?
Current submarine not support multi-user. So fix workbench notebook page to just use "default" namespace.

### What type of PR is it?
[Improvement]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-631

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/729541287

### Screenshots (if appropriate)
螢幕錄製 2020-09-23 下午3

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@lowc1012  @jiwq  @pingsutw  @xunliu  Please help me review this fronted page. Thanks :)

**Review by jiwq — APPROVED** [review]
+1 Thanks @kobe860219 for the work.

File: submarine-workbench/workbench-web-ng/src/app/pages/workbench/notebook/notebook.component.ts [hunk 1]
```diff
@@ -269,10 +255,12 @@ export class NotebookComponent implements OnInit {
       }
     }
 
+    console.log(newNotebookSpec);
```
Comment (xunliu) [inline submarine-workbench/workbench-web-ng/src/app/pages/workbench/notebook/notebook.component.ts:258]: You can comment out this output log and open it when debugging?

Comment (kobe860219) [reply submarine-workbench/workbench-web-ng/src/app/pages/workbench/notebook/notebook.component.ts]: @xunliu  Thanks your suggestion. I have comment out that output log.

**Review by pingsutw — APPROVED** [review]
+1

**Comment by lowc1012** [comment]
Thanks @kobe860219. It looks good to me.

**Review by xunliu — APPROVED** [review]
@kobe860219 Thank you for contribution!
LGTM

**Pull request closed without merging**